modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -4,19 +4,17 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="network-observability-filtering-ebpf-rule_{context}"]
-= Filtering eBPF flow data using a global rule
-You can configure the `FlowCollector` to filter eBPF flows using a global rule to control the flow of packets cached in the eBPF flow table.
+= Filtering eBPF flow data
+You can configure the `FlowCollector` to filter eBPF flows using multiple rules and features to control the flow of packets cached in the eBPF flow table.
 
 .Procedure
 . In the web console, navigate to *Operators* -> *Installed Operators*.
 . Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
 . Select *cluster*, then select the *YAML* tab.
-. Configure the `FlowCollector` custom resource, similar to the following sample configurations:
+. Configure the `FlowCollector` custom resource, similar to the following sample configuration:
++
+.Example `FlowCollector` YAML with multiple rules
 +
-
-[%collapsible]
-.Filter Kubernetes service traffic to a specific Pod IP endpoint
```

Comment:
should not be removed but also need to mention about PAcketDrop

Reply:
Filter Kubernetes service traffic to a specific Pod IP endpoint and service flows with Drops